core/whistle_couch-1.0.0/src/couch_util.erl

```diff
@@ -81,6 +81,12 @@
 -define(MAX_BULK_INSERT, 2000).
 -define(RETRY_504(F), retry504s(fun() -> F end)).
 
+-define(PUBLISH_FIELDS, [<<"pvt_type">>
+                         ,<<"pvt_account_id">>
+                         ,<<"pvt_created">>
+                         ,<<"pvt_modified">>
```

Comment:
Maybe we default these to `wh_util:current_tstamp` when building the AMQP payload?